core/scm.js

```diff
@@ -251,7 +251,22 @@ const SCHEMA_HOOK = Joi.object()
         releaseAuthor: Joi.string()
             .allow('')
             .optional()
-            .label('Author of the event')
+            .label('Author of the event'),
+
+        addedFiles: Joi.array()
+            .items(Joi.string().allow(''))
+            .optional()
+            .label('Added files of head commit'),
+    
+        modifiedFiles: Joi.array()
+            .items(Joi.string().allow(''))
+            .optional()
+            .label('Modified files of head commit'),
+    
+        removedFiles: Joi.array()
```

Comment:
`deleted` is used in git command, so I think that it is better to use `deleted` instead of `removed`.

https://git-scm.com/docs/git-status#_output

Reply:
The GitHub and GitLab webhook specification treats deleted files as `removed`, so I think it's better to leave them as `removed`.

https://docs.github.com/ja/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push

Reply:
Okey, I see. Thank you.